app/models/report.rb

```diff
@@ -51,6 +51,14 @@ def destroy_reported_item
     mark_as_reviewed
   end
 
+  def mark_as_nsfw
```

Comment:
>but if not you can spend either contacting ten different people or mark it quickly as nsfw

maybe it's time to set a community of volunteers for joindiaspora ? :) We now have the moderator role, just for that.